recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
     - sigstksz_gcc11.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
```suggestion
  number: {{ upstream_build_num + cf_build_num }}
```

Reply:
Do you know if the bot will reset this to zero, or will it leave this alone?

Reply:
I looked around and based on the documentation in https://conda-forge.org/docs/maintainer/knowledge_base.html#including-a-no-mpi-build and the examples in https://github.com/conda-forge/hdf5-feedstock/pull/168/commits/91ec741b622b3df8203b20d0d233596d98cd3518#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL4 and  https://github.com/conda-forge/libsharp-feedstock/pull/11/commits/0eacd9c5f4b0a73e40f49b0423fce9c787f7c79d#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL4 it looks like the bot actually will do the right thing for us if we rename `cf_build_num` to just `build`. What do you think?